src/opendr/perception/object_detection_2d/gem/gem_learner.py

```diff
@@ -0,0 +1,1059 @@
+
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I believe you chopped off part of the license by mistake, the years are missing for instance.